src/elements/content-sidebar/activity-feed/comment/__tests__/CreateReply.test.js

```diff
@@ -0,0 +1,119 @@
+// @flow
+
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { ContentState, EditorState } from 'draft-js';
+import CreateReply from '../CreateReply';
+import localize from '../../../../../../test/support/i18n';
+import messages from '../messages';
+
+jest.mock('react-intl', () => ({
+    ...jest.requireActual('react-intl'),
+    FormattedMessage: ({ defaultMessage }: { defaultMessage: string }) => <span>{defaultMessage}</span>,
+}));
+
+const onCancel = jest.fn();
+const onFocus = jest.fn();
+const onSubmit = jest.fn();
+const onClick = jest.fn();
+
+const getWrapper = props =>
+    render(
+        <IntlProvider locale="en">
+            <CreateReply
+                isDisabled={false}
+                onCancel={onCancel}
+                onFocus={onFocus}
+                onSubmit={onSubmit}
+                onClick={onClick}
+                showReplyForm={false}
+                {...props}
+            />
+        </IntlProvider>,
+    );
+
+describe('elements/content-sidebar/ActivityFeed/comment/CreateReply', () => {
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
+    const replyMessage = localize(messages.reply.id);
+    const replyInThreadMessage = localize(messages.replyInThread.id);
+
+    test('should correctly render CreateReply button', () => {
+        getWrapper();
+
+        expect(screen.getByText(replyMessage)).toBeVisible();
+        expect(screen.queryByText(replyInThreadMessage)).not.toBeInTheDocument();
+    });
+
+    test('should correctly render CreateReply form', () => {
+        getWrapper({ showReplyForm: true });
+
+        expect(screen.getByText(replyInThreadMessage)).toBeVisible();
+        expect(screen.queryByText(replyMessage)).not.toBeInTheDocument();
+    });
+
+    test('should disable Reply button if isDisabled property is true', () => {
+        getWrapper({ isDisabled: true });
+
+        const replyButton = screen.getByRole('button', { name: replyMessage });
+        expect(replyButton).toHaveAttribute('aria-disabled', 'true');
+
+        fireEvent.click(replyButton);
+        expect(onClick).not.toBeCalled();
+    });
+
+    test('should call onClick when reply button is clicked', () => {
+        getWrapper();
+
+        fireEvent.click(screen.getByText(replyMessage));
+        expect(onClick).toBeCalledTimes(1);
+        expect(onSubmit).not.toBeCalled();
+        expect(onFocus).not.toBeCalled();
+        expect(onCancel).not.toBeCalled();
+    });
+
+    test('should not show form when isDisabled', () => {
```

Comment:
minor: `should not show form when isDisabled` -> `should not show form when isDisabled is true`